src/shims/ApiCompatBaseline.netcoreapp.netfx461.txt

```diff
@@ -15,6 +13,25 @@ MembersMustExist : Member 'System.CodeDom.Compiler.CompilerParameters.Evidence.g
 MembersMustExist : Member 'System.CodeDom.Compiler.CompilerParameters.Evidence.set(System.Security.Policy.Evidence)' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'System.CodeDom.Compiler.CompilerResults.Evidence.get()' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'System.CodeDom.Compiler.CompilerResults.Evidence.set(System.Security.Policy.Evidence)' does not exist in the implementation but it does exist in the contract.
+Compat issues with assembly System.Configuration:
+CannotRemoveBaseTypeOrInterface : Type 'System.Configuration.CallbackValidatorAttribute' does not implement interface 'System.Runtime.InteropServices._Attribute' in the implementation but it does in the contract.
```

Comment:
Can you please move all the ones we'll never fix into ApiCompatBaseline.netcoreapp.netfx461.ignore.txt - ie the ones deriving from the underscore interfaces